integrated_channels/sap_success_factors/utils.py

```diff
@@ -253,7 +253,8 @@ def resolve_removed_courses(self, previous_audit_summary):
                     'sap_success_factors',
                     'SAPSuccessFactorsGlobalConfiguration'
                 ).current().provider_id,
-                'launchURL': get_course_track_selection_url(x['enterprise_customer'], x['key']),
+                'launchURL': (get_enterprise_course_enrollment_url(x['enterprise_customer'], x['key']) or
+                              get_course_track_selection_url(x['enterprise_customer'], x['key'])),
```

Comment:
I've actually thought of making this a setting for the Enterprise Customer, if some Enterprises might want the track selection page instead... but I suppose an overarching flag would make more sense for transitioning in case the page isn't ready. We can talk more Monday.